### DISCUSSION TITLE
integrations: image data demo

### DISCUSSION THREAD

**Discussion by luisoala** [mlcommons/croissant#77] [General]
hi gang, is there activity for a demo template with image data (scanning /datasets i had impression mainly tabular)? if no who is interested? i have some demo implementations in lance and activeloop deeplake for image data that we could try to make croissant compliant as integration demos.

**Comment by pierrot0** [comment]
Hi Luis,

PASS is a dataset with images: https://github.com/mlcommons/croissant/blob/main/datasets/pass/metadata.json.
This config uses the original packaging of the dataset, ie: image files in tar files + CSV.

If you know datasets stored using lance or activeloop deeplake, I think it would indeed be valuable to have examples of datasets reading from those formats/stores too!
